rest-api/model/questionnaire_response.py

```diff
@@ -53,7 +54,7 @@ class QuestionnaireResponseAnswer(Base):
   valueDecimal = Column('value_decimal', Float)
   valueInteger = Column('value_integer', Integer)
   # Is this big enough?
-  valueString = Column('value_string', String(1024))
+  valueString = Column('value_string', String(MAX_MYSQL_VARCHAR))
```

Comment:
Let's try changing this to Text.

Either way, once you've made a model change like this, follow these steps:

1) From rest-api dir, while server is running, run 'tools/setup_local_database.sh'
2) From client dir, run './run_client.sh generate_fake_data.py --num_participants 10'. this should generate fake data for 10 participants. You can look in questionnaire_response_answer using mysql client to confirm rows with value_string set were inserted.
3) From rest-api dir, run tools/generate_schema.sh. This will make a new schema migration in alembic/versions. Make sure it looks reasonable and git add it.
4) From rest-api dir, run tools/upgrade_schema.sh. This will apply the migration you just created; check the database afterwards with mysql client to confirm the data was preserved but the column is bigger now.